docs/conf.py

```
@@ -7,7 +7,7 @@
 
 # Project information
 project = "pyseekdb"
-copyright = "2025, OceanBase"
+copyright = "2025, OceanBase"  # noqa: A001
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `A001`)

Remove unused `noqa` directive

(RUF100)